poetry_to_pre_commit/sync_hooks_additional_dependencies.py

```diff
@@ -86,14 +117,19 @@ def sync_hook_additional_deps(
             for group in groups:
                 deps.update(deps_by_group.get(group, set()))
 
-            hook["additional_dependencies"] = sorted(deps)
+            hook["additional_dependencies"] = sorted(
+                update_or_remove_additional_deps(deps, hook["additional_dependencies"])
```

Comment:
(whenever a (nontrivial) function has 2 or more params, I prefer using kwargs if you don't mind. Makes reading the code much easier.)
(also, the line is getting long... let's split it?)

Reply:
> I prefer using kwargs

Alright - done.

> the line is getting long... let's split it?

This is no job for a human being :-p This can you can configured in `pyproject.toml` with e.g.:
```
[tool.ruff]
line-length = 100
```
I can open a separate PR for that if you like, just tell me what value you would like to use.

Reply:
No, I meant creating an intermediate variable. I think ruff doesn't do that (yet) :D

Reply:
I see! I tried different things (using an intermediate variable, doing the assignment below `if... else`, defining a function in the local scope...), but I am not satisfied with any of them :-/ Maybe a function to process the hook and get 2 indentation levels back? If you don't mind, I prefer to leave that to you, so you can directly pick your favorite option.